tsdb/config.go

```diff
@@ -63,6 +71,28 @@ type Config struct {
 	WALFlushColdInterval      toml.Duration `toml:"wal-flush-cold-interval"`
 	WALPartitionSizeThreshold uint64        `toml:"wal-partition-size-threshold"`
 
+	// WAL configuration options for tsm1 introduced in 0.9.5
+	WALFlushMemorySizeThreshold int `toml:"wal-flush-memory-size-threshold"`
+	WALMaxMemorySizeThreshold   int `toml:"wal-max-memory-size-threshold"`
+
+	// compaction options for tsm1 introduced in 0.9.5
+
+	// IndexCompactionAge specifies the duration after the data file creation time
+	// at which it is eligible to be compacted
+	IndexCompactionAge time.Duration `toml:"index-compaction-age"`
+
+	// IndexMinimumCompactionInterval specifies the minimum amount of time that must
+	// pass after a compaction before another compaction is run
+	IndexMinimumCompactionInterval time.Duration `toml:"index-minimum-compaction-interval"`
+
+	// IndexCompactionFileCount specifies the minimum number of data files that
+	// must be eligible for compaction before actually running one
+	IndexCompactionFileCount int `toml:"index-compaction-file-count"`
```

Comment:
For consistency, it seems this variable should be named `IndexMinimumCompactionFileCount`. If these variable names are getting too long for taste, then just use `Min` in both, but since they a both minimum conditions, naming like this is a big help.